e3db/client.py

```diff
@@ -178,18 +177,18 @@ def __encrypt_record(self, plaintext_record):
             self.__put_access_key(writer_id, user_id, self.client_id, record_type, ak)
 
         # Loop through the plaintext fields and encrypt them
-        for key, value in record['data'].iteritems():
+        for key, value in record['data'].items():
             dk = Crypto.random_key()
             efN = Crypto.random_nonce()
-            ef = Crypto.encrypt_secret(dk, str(value), efN)
+            ef = Crypto.encrypt_secret(dk, value, efN)
             # remove nonce from ciphertext
             ef = ef[len(efN):]
             edkN = Crypto.random_nonce()
             edk = Crypto.encrypt_secret(ak, dk, edkN)
             # remove nonce from ciphertext
             edk = edk[len(edkN):]
 
-            record['data'][key] = ".".join([Crypto.base64encode(c) for c in [edk, edkN, ef, efN]])
+            record['data'][key] = ".".join([Crypto.base64encode(c).decode("utf-8") for c in [edk, edkN, ef, efN]])
```

Comment:
Curious why this one specified "utf-8" while line 297 just uses the `.decode()` method?

Reply:
decode without explicit params is "utf-8" I should make these all match. I will make them all explicit